news/new2.rst

```diff
@@ -0,0 +1,3 @@
+**Changed:**
+
+* new 999-load2.py
```

Comment:
I think the news is better if you don't leave a blank line, but you do leave a blank space before the news item.  This news item is not very informative.  It doesn't say what the changes are.  Please can you make it better?  It forms important parts of the documentation.